libraries/AP_AHRS/AP_AHRS_Backend.h

```diff
@@ -24,6 +24,7 @@
 #include <inttypes.h>
 #include <AP_Airspeed/AP_Airspeed.h>
 #include <AP_InertialSensor/AP_InertialSensor.h>
+#include <AP_NavEKF/AP_Nav_Common.h>
```

Comment:
Commit message is not correct for this change

Reply:
fixed